source/ngdocs/formatForPFsite.js

```diff
@@ -0,0 +1,15 @@
+setTimeout(function(){
+    $( ".improve-docs" ).hide();
```

Comment:
It looks like you're using tabs to indent?  If so, please use spaces instead per http://codeguide.patternfly.org/#html-syntax